common/src/main/java/com/sequenceiq/cloudbreak/validation/ValidationResult.java

```diff
@@ -11,27 +11,37 @@
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 
+import org.apache.commons.lang3.StringUtils;
+
 public class ValidationResult {
 
     private final State state;
 
     private final List<String> errors;
 
-    private final String formattedErrors;
+    private String formattedErrors;
+
+    private final String prefix;
 
-    private ValidationResult(State state, SortedSet<String> errors) {
+    private ValidationResult(State state, SortedSet<String> errors, String prefix) {
         this.state = state;
         this.errors = new ArrayList<>(errors);
-        formattedErrors = IntStream.range(0, this.errors.size())
+        this.prefix = prefix;
+        if (!StringUtils.isEmpty(prefix)) {
+            formattedErrors = prefix + ": \n";
+        } else {
```

Comment:
I think the else clause is not needed.

Reply:
Unfortunately, the else clause is needed because if formattedErrors  is null, the `null` will appear the start of the formatted errors